src/types/permission-types.ts

```diff
@@ -68,25 +68,34 @@ export type PermissionRevocationData = {
 /**
  * The data model for a permission scope.
  */
-export type PermissionScope = {
-  interface: DwnInterfaceName;
-  method: DwnMethodName;
-} | RecordsPermissionScope;
+export type PermissionScope = ProtocolPermissionScope | MessagesPermissionScope | EventsPermissionScope | RecordsPermissionScope;
+
+export type ProtocolPermissionScope = {
+  interface: DwnInterfaceName.Protocols;
+  method: DwnMethodName.Configure | DwnMethodName.Query;
+};
+
+export type MessagesPermissionScope = {
+  interface: DwnInterfaceName.Messages;
+  method: DwnMethodName.Get;
```

Comment:
NOTE: Will add `protocol: string;` to this scope when it is implemented in a subsequent PR.